### PR TITLE
Docs first pass

### PR DESCRIPTION
**Pull request by lu-zero**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -2,6 +2,8 @@
 //!
 //! Provides all the building blocks to serve [Web Of Things](https://www.w3.org/WoT/) Things.
 
+#![warn(missing_docs)]
```
Comment (dodomorandi) [inline src/lib.rs]: Not a criticism, just a thought: if you leave this, the CI will fail from now on if we omit a doc. Do we want this?

File: src/servient/builder.rs [hunk 2]
```diff
@@ -16,9 +16,10 @@ use wot_td::{
 };
 
 #[doc(hidden)]
-/// ThingBuilder ExtendableThing used to build a Servient via HttpRouter methods.
+/// ThingBuilder ExtendableThing used to build a Servient
 ///
-/// It is not needed to know about it nor use it.
+/// It is not needed to know about it nor use it directly.
+/// Instantiate a correct builder by calling [Servient::builder].
```
Comment (dodomorandi) [inline src/servient/builder.rs]:
```suggestion
/// Instantiate a correct builder by calling [`Servient::builder`].
```

File: src/servient/builder.rs [hunk 3]
```diff
@@ -78,12 +79,15 @@ where
     }
 }
 
-/// Trait extension to build a [Servient] from an extended [ThingBuilder]
+/// Trait extension to build a [`Servient`] from an extended [`ThingBuilder`]
```
Comment (dodomorandi) [inline src/servient/builder.rs:82]: We also need the following:
```rust
/// [`ThingBuilder`]: wot_td::builder::ThingBuilder
```

**Review by dodomorandi — CHANGES_REQUESTED** [review]
Just some minor considerations :relaxed:

File: src/servient/builder.rs (hunk 2 above)
Comment (dodomorandi) [reply src/servient/builder.rs:22]: Nope, my fault :sweat:

File: src/servient/builder.rs (hunk 3 above)
Comment (dodomorandi) [reply src/servient/builder.rs:82]: Nope, my fault :sweat:

File: src/lib.rs (hunk 1 above)
Comment (lu-zero) [reply src/lib.rs]: Probably it can wait until we reach 1.0.

File: src/servient/builder.rs (hunk 2 above)
Comment (lu-zero) [reply src/servient/builder.rs]: Folded in the changes

**Pull request merged**